client/src/pages/Contest/NewContestForm/NewContestForm.tsx

```diff
@@ -0,0 +1,220 @@
+import { useState } from 'react';
+import { FunctionComponent } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import Box from '@material-ui/core/Box';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+import Grid from '@material-ui/core/Grid';
+import { CircularProgress } from '@material-ui/core';
+import PreferedDesigns from '../PreferedDesigns/PreferedDesigns';
+
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    {
+      title,
+      description,
+      prizeAmount,
+      date,
+      time,
+      timezone,
+    }: {
+      title: string;
+      description: string;
+      prizeAmount: string;
+      date: string;
+      time: string;
+      timezone: string;
+    },
+    {
+      setSubmitting,
+    }: FormikHelpers<{
+      title: string;
+      description: string;
+      prizeAmount: string;
+      date: string;
+      time: string;
+      timezone: string;
+    }>,
+  ) => void;
+}
+
+const NewContestForm: FunctionComponent<Props> = ({ handleSubmit }: Props): JSX.Element => {
+  const classes = useStyles();
+  const [selectedDate, setSelectedDate] = useState<Date | null>(new Date());
+
+  const handleDateChange = (date: Date | null) => {
+    setSelectedDate(date);
+  };
+
+  const times = [
+    '12:00 am',
+    '1:00 am',
+    '2:00 am',
+    '3:00 am',
+    '4:00 am',
+    '5:00 am',
+    '6:00 am',
+    '7:00 am',
+    '8:00 am',
+    '9:00 am',
+    '10:00 am',
+    '11:00 am',
+    '12:00 pm',
+    '1:00 pm',
+    '2:00 pm',
+    '3:00 pm',
+    '4:00 pm',
+    '5:00 pm',
+    '6:00 pm',
+    '7:00 pm',
+    '8:00 pm',
+    '9:00 pm',
+    '10:00 pm',
+    '11:00 pm',
+  ];
+
+  return (
+    <Formik
+      initialValues={{
+        title: '',
+        description: '',
+        prizeAmount: '',
+        date: '',
+        time: '',
+        timezone: '',
+      }}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <Grid container direction="row" justify="center" spacing={4}>
+            <Grid item xs={11} md={9}>
+              <label htmlFor="title">What do you need designed?</label>
```

Comment:
You can use the MUI FormLabel component here :) 

Reference: https://material-ui.com/api/form-label/